### DISCUSSION TITLE
Add a Blynk server ?

### DISCUSSION THREAD

**Discussion by Nico83500** [swizzin/swizzin#431] [Feature Requests] [answered]
Hi,
I use Swizzin and I would like to add a Blynk server. I just installed java 11 and launch server.jar, that's all.
I also need to use Letsencrypt certificates so I copy it from /etc/nginx/ssl/<domain.name> to my Blynk folder but is it possible to use it directly (I have an issue with folder permissions).
Or more simply, is it possible to add Blynk to Swizzin ?
Thank you !

**Comment by liaralabs** [comment, marked as answer]
Hey Nico,

It's not advisable to copy certs out of the directory unless you are going to also run a cronjob that will copy them every time the certificate is updated. LE certs only last for 3 months, so without some method of keeping the new cert up to date, you'll end up with an expired cert sooner or later.

By design the `/etc/nginx/ssl` folder has permissions `700` which means only root can access this directory.

If you're the only user on the server there is less risk in opening up the permissions on this directory (i.e. `755`). Alternatively, you can use groups to provide read access only to one specific group (and then you can add the group the Blynk server user).

**Comment by flying-sausages** [comment]
> is it possible to add Blynk to Swizzin

I think the use-case of this is a bit beyond a _seedbox_ toolset
